circuits/lib/utils/claimUtils.circom

```diff
@@ -266,3 +266,33 @@ template getClaimExpiration() {
 	}
 	expiration <== expirationBits.out;
 }
+
+// getSubjectLocation extract subject from claim flags.
+template getSubjectLocation() {
+    signal input claimFlags[32];
+    signal output out;
+
+    component subjectBits = Bits2Num(3);
+
+    for (var i=0; i<3; i++) {
+        subjectBits.in[i] <== claimFlags[i];
+    }
+
+    out <== subjectBits.out;
+}
+
+// isExpiration return 1 if expiration flag is set otherwise 0.
+template isExpiration() {
```

Comment:
Should be `hasExpiration` or `isExpirable`

Reply:
Same for file names containing tests of this template.

Reply:
agree with `isExpirable`